resources/js/Pages/Results.vue

```diff
@@ -267,4 +310,25 @@ h2 {
     margin-top: $dimension-layout-xsmall;
 }
 
+p.list-intro {
+    margin-bottom: 0
+}
+
+#description-section {
+    position:relative;
```

Comment:
I'm not sure we need this relative positioning here.

Reply:
That's right, I think that line remained from a previous, slightly different attempt to place the button.